examples/testprograms/twilio/main.star

```diff
@@ -0,0 +1,16 @@
+load("env", "TWILIO_ACCOUNT_SID", "TWILIO_AUTH_TOKEN", "TWILIO_MSG_SERVICE_SID")
+load("libs/integrations/twilio.star", "twilio")
```

Comment:
since this is a predefined lib, 
I think we can try (in a different ticket in the future) to add "libs" or "plugins" section to the manifest, and inject this code automatically 

so
```yaml
project:
  libs:
    - name: twilio
      source: github/.... (optional, only if name is not auto resolved by ak core)
       data:
         account_sid: (from secret)
         twilio_auth_token: (from secret)
```

then the script can just declare what it needs and start using it
without loading env and lib and setting up a client (although it can)